src/routes/organization.js

```diff
@@ -43,8 +45,18 @@ router.post('/', loginRequired, async (ctx) => {
   if (!config.mail.allowTest && isTestEmail(ctx.request.body.email)) {
     throw new ValidationError('Invalid email')
   }
+
+  const referredById = ctx.request.body?.referredBy
+  if (referredById) {
+    const user = await User.get(referredById)
+
+    if (!user.identity_verified)
```

Comment:
@jeyem Is this fine or better create a function `User.getVerified` to move this check to database layer?